src/main/java/games/strategy/triplea/settings/SettingsWindow.java

```diff
@@ -59,16 +70,18 @@ public void open() {
     }
   }
 
-  private static JComponent createContents(final Runnable closeListener) {
-    final JTabbedPane tabbedPane = SwingComponents.newJTabbedPane(1000, 400);
-
-    Arrays.stream(SettingType.values()).forEach(settingType -> {
-      final List<ClientSettingSwingUiBinding> settings = getSettingsByType(settingType);
-
-      final JComponent tab = buildTab(settings, closeListener);
-      tabbedPane.add(settingType.tabTitle, tab);
+  private JComponent createContents() {
+    tabbedPane = SwingComponents.newJTabbedPane(1000, 400);
+    settingTypes.forEach(settingType -> {
+      tabbedPane.add(settingType.tabTitle, buildTabPanel(getSettingsByType(settingType)));
```

Comment:
I assume this wasn't inlined because of line width

Reply:
Nope.  It was originally a multi-line lambda, and I simply deleted all but one line and didn't think to collapse it. :smile:  But, yes, it would still need to be wrapped to a new line because it's too long.